libreyolo/models/depth_anything/_vendor/dpt.py

```
@@ -0,0 +1,221 @@
+        path_2 = self.scratch.refinenet2(path_3, layer_2_rn, size=layer_1_rn.shape[2:])
+        path_1 = self.scratch.refinenet1(path_2, layer_1_rn)
+        
+        out = self.scratch.output_conv1(path_1)
+        out = F.interpolate(out, (int(patch_h * 14), int(patch_w * 14)), mode="bilinear", align_corners=True)
+        out = self.scratch.output_conv2(out)
+        
+        return out
+
+
+class DepthAnythingV2(nn.Module):
+    def __init__(
```

Ruff (0.15.18)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)